### PR TITLE
Exact version lock file

### LINKED ISSUE #27: Exact versions: why
Hey!

Amazing work on the package. Just wanted to check: why is the preferred approach to use exact versions?

Personally I've never seen this being a recommendation. I've actually seen the opposite - this being disapproved. So I'm genuinely curious.

Source: https://github.com/wallet77/qualscan/blob/main/doc/dependencies-exact-version.md

### PR DESCRIPTION
**Pull request by wallet77**

### CODE REVIEW

**Comment by wallet77** [comment]
@MatissJanis 

This PR should do the trick for issue #28

File: src/plugins/dependencies-exact-version/cmd.js [hunk 1]
```diff
@@ -10,10 +11,14 @@ const semver = require('semver')
  * @param {String} key : defines if we are focusing on dependencies or devDependencies
  */
 const checkVersion = (badVersions, key) => {
-    for (const dependency in global.packagefile[key]) {
-        const depVersion = (global.packagefile[key][dependency]).replace('workspace:', '')
-        if (!semver.valid(depVersion)) {
-            badVersions[key][dependency] = depVersion
+    // only check version in package.json
+    // if no package-lock.json was found
+    if (!fs.existsSync(path.join(process.env.QUALSCAN_PROJECT_PATH, 'package-lock.json'))) {
```
Comment (MatissJanis) [inline src/plugins/dependencies-exact-version/cmd.js]: Would also be good to check for `yarn.lock` file.

Comment (wallet77) [reply src/plugins/dependencies-exact-version/cmd.js]: Is it better now?

**Review by MatissJanis — APPROVED** [review]
Amazing! 🎉

**Comment by wallet77** [comment]
@MatissJanis shipped with version 2.6.0

**Pull request merged**